### PR TITLE
docs: unify plugin Description

### LINKED ISSUE #6477: docs: zh/*.md some file first dir is "名字"
### Improved or typo fixed.

like https://apisix.apache.org/zh/docs/apisix/plugins/key-auth
image

Would it be better to  "简介"？or "描述"

### Solution

名称->简介

### PR DESCRIPTION
**Pull request by kwanhur**
### What this PR does / why we need it:

Unify first title in plugins' docs, `Description` for en and `描述` for zh.

Fix #6477

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by zaunist — APPROVED** [review]
LGTM

File: docs/zh/latest/plugins/zipkin.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ title: zipkin
 #
 -->
 
-## 名字
+## 描述
 
 [Zipkin](https://github.com/openzipkin/zipkin) 是开源的分布调用链追踪系统。该插件基于[Zipkin API规范](https://zipkin.io/pages/instrumenting.html)，支持收集跟踪信息，并上报 Zipkin Collector。
```
Comment (hf400159) [inline docs/zh/latest/plugins/zipkin.md:26]:
```suggestion
[Zipkin](https://github.com/openzipkin/zipkin) 是开源的分布调用链追踪系统。该插件基于[Zipkin API规范](https://zipkin.io/pages/instrumenting.html)，支持收集跟踪信息，并上报 Zipkin Collector。
```

**Review by hf400159 — DISMISSED** [review]
LGTM

**Review by LiteSun — APPROVED** [review]

**Comment by spacewander** [comment]
@juzhiyuan 
Please take a look

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Review by leslie-tsang — APPROVED** [review]

**Pull request merged**